src/http.ts

```diff
@@ -99,7 +99,10 @@ export async function getIpFamily(authHost: string): Promise<IpFamily> {
     },
     0,
   );
-  return res.success ? family : undefined;
+
+  const ipv6Incompatible = (<FailedResponse>res).errorCode == ErrorCodes.dnsNotFound;
```

Comment:
why the shallow comparison? (`==`)

Reply:
Knew that these two are always of number type. Changed it to strict equality to be future-proof here ✅